src/app/pages/signin/store/signin.store.ts

```diff
@@ -35,13 +35,15 @@ interface SigninState {
     disabledReasons?: FailoverDisabledReason[];
   };
   loginBanner: string;
+  queryToken: string | null;
 }
 
 const initialState: SigninState = {
   isLoading: false,
   wasAdminSet: true,
   failover: null,
   loginBanner: null,
+  queryToken: null,
```

Comment:
I don't think it needs to be part of state here and have a selector. It's not used outside of the component.